helpers/helpers.py

```diff
@@ -20,8 +20,160 @@
 import config.config as cfg
 import discord
 import requests
+import subprocess
+import os
 from typing import Optional, Dict, Any, List, Tuple
 from logger.logger import logger
+from web3 import Web3
+
+
+def modify_space_settings(quorum_value):
+    """
+    Submit a command to modify the space settings on Snapshot.
+    Called from tasks/tasks.py.
```

Comment:
dont think we need a comment on where it's called from. could change in the future then this comment will be outdated

Reply:
> dont think we need a comment on where it's called from. could change in the future then this comment will be outdated

Nws, removed that part.